test.html

```diff
@@ -1,162 +1,207 @@
-               <ul class="ul2">
-                    <li><a href=""><i class="fa-regular fa-magnifying-glass"></i></a></li>
-                    <li><a href="">Sign in</a></li> /
-                    <li><a href="">Sign up</a></li>
-                    <li><a href="" class="free_btn">Start a free project</a></li>
-                </ul>
-            </div>
-        </nav>
-        <hr>
+    <link rel="stylesheet" href="test.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="bribbble.html"><img id="logo" src="images/logo.jpg" /></a>
+        <div class="menu" id="menuLink">
+          <ul class="ul1">
+            <li><a href="" class="nav_btn">Inspiration</a></li>
+            <li><a href="" class="nav_btn">Find work</a></li>
+            <li><a href="" class="nav_btn">Learn Design</a></li>
+            <li><a href="" class="nav_btn">Go Pro</a></li>
+            <li><a href="" class="nav_btn">Marketplace</a></li>
+            <li><a href="" class="nav_btn">Hire Designers</a></li>
+          </ul>
+          <i class="fa-solid fa-xmark" id="close-menu"></i>
+        </div>
+        <i class="fa-solid fa-bars" id="open-menu"></i>
+        <div class="sign_menu">
+          <ul class="ul2">
+            <li>
+              <a href=""><i class="fa-solid fa-magnifying-glass"></i></a>
+            </li>
+            <li><a href="">Sign in</a></li>
+            /
+            <li><a href="">Sign up</a></li>
+            <li><a href="" class="free_btn">Start a free project</a></li>
+          </ul>
+        </div>
+      </nav>
+      <hr />
     </header>
-    <br><br><br>
+
     <section class="page1">
-        <div class="grid_box">
-            <div class="pg1_box1"></div>
-            <div class="pg1_box2">
-                <a href=""><img class="masking1" src="Images\masking.png" alt=""></a>
-                <h1>Masking and Clipping</h1>
-                <ul class="pg1_ul1">
-                    <li><a href="">TorusKit</a></li>
-                    <li><a href="">.</a></li>
-                    <li><a href="">Follow</a></li>
-                </ul>
-                <br>
-                <ul class="pg1_ul2">
-                    <li><a href="" class="save_btn">Save</a></li>
-                    <li><a href="" class="like_btn"><i class="fa-solid fa-heart"></i> Like</a></li>
-                </ul>
-            </div>
-            <div class="pg1_box3"></div>
-            <div class="pg1_box4">
-                <video src="Images\animated_small20191213-15576-1092sb.mp4" autoplay loop></video>
-                <br>
-                <p>We're working on circular clipping that will mask the inner element and make it<br> non-interactive.After the certain action
-                (in this case it's trigger-hover), the mask <br> will grow and show the content. It's a simple th:mask-circular markup.
-                <br>Design is part of upcoming homepage redesign:)</p>
-                <br>
-                <ul class="pg1_ul3">
```

Comment:
This entire section in full width view is overlapping the text above. Please double check its positioning